fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -552,13 +553,13 @@ struct EncryptedRangeFileWriter : public IRangeFileWriter {
 		constexpr static FileIdentifier file_identifier = 3152016;
 
 		// TODO: Compression is not currently supported so this should always be false
-		bool compressionEnabled = false;
+		bool configurableEncryptionEnabled = false;
```

Comment:
any reason to repurpose the flag? Especially given they are completely different? Also, `configurable_encryption` might be removed once all changes are stabilized.

Reply:
I dont think we'll ever support compression and if we do we can add it again, I dont think there is much purpose in keep this flag. I think its better to keep `configurableEncryptionEnabled` while we're running both

Reply:
Sounds good. We can then remove the flag once configurable_encryption is supported by default. Please update the comment as is still reads `Compression`